src/main/plugins/org.dita.base/build_preprocess_template.xml

```diff
@@ -487,7 +506,7 @@ See the accompanying LICENSE file for applicable license.
           filedirparameter="FILEDIR">
         <ditaFileset format="dita" processingRole="normal"/>
         <param name="TRANSTYPE" expression="${transtype}"/>
-        <param name="FILTERFILEURL" expression="${dita.input.filterfile.url}"/>
+        <param name="FILTERFILEURL" expression="${mergedditaval.url}"/>
```

Comment:
This is not backwards compatible in cases where users still assume `dita.input.filterfile.url`is available. Why not just keep on using the same old property, just point it to the merged filter file?